src/student_registry_project/ex_1.rs

```diff
@@ -0,0 +1,384 @@
+    
+// }
+fn get_courses(rust_cohort: &mut StudentRegistry) -> u32 {
+    print!("{:#?}", rust_cohort.course_registry);
+    io::stdout().flush().unwrap();
+
+    print!("Enter course id: \n");
+    io::stdout().flush().unwrap();
+    let mut course_id = String::new();
+    io::stdin()
+        .read_line(&mut course_id)
+        .expect("Failed to read line");
+    let course_id = course_id
+        .trim()
+        .parse::<u32>()
+        .expect("Invalid input. Please enter a number.");
+    // let course = rust_cohort.get_course_by_id(course_id);
+    // println!("course: {course:#?}");
+    course_id
+}
+
+fn student_flow(rust_cohort: &mut StudentRegistry) {
+    let mut student_state: i32 = 0;
+    loop {
+        let mut input = String::new();
+
+        print!("1. Add student\n2. Get student\n3.Enroll student in course\n4.Return\n");
+        io::stdout().flush().unwrap();
+
+        io::stdin()
+            .read_line(&mut input)
+            .expect("Failed to read line");
+        student_state = input
+            .trim()
+            .parse::<i32>()
+            .expect("Invalid input. Please enter a number.");
+
+        // Add student
+        if student_state == 1 {
+            print!("Enter first name: ");
+            io::stdout().flush().unwrap();
+            let mut first_name = String::new();
+            io::stdin()
+                .read_line(&mut first_name)
+                .expect("Failed to read line");
+            let first_name = first_name.trim().to_string();
+
+            print!("Enter last name: ");
+            io::stdout().flush().unwrap();
+            let mut last_name = String::new();
+            io::stdin()
+                .read_line(&mut last_name)
+                .expect("Failed to read line");
+            let last_name = last_name.trim().to_string();
+
+            print!("Enter age: ");
+            io::stdout().flush().unwrap();
+            let mut age = String::new();
+            io::stdin()
+                .read_line(&mut age)
+                .expect("Failed to read line");
+            let age = age
+                .trim()
+                .parse::<u8>()
+                .expect("Invalid input. Please enter a number.");
+
+            print!("Enter height: ");
+            io::stdout().flush().unwrap();
+            let mut height = String::new();
+            io::stdin()
+                .read_line(&mut height)
+                .expect("Failed to read line");
+            let height = height
+                .trim()
+                .parse::<f32>()
+                .expect("Invalid input. Please enter a number.");
+
+            print!("Enter sex: 1 for male, 2 for female: ");
+            let mut sex = String::new();
+            io::stdout().flush().unwrap();
+            io::stdin()
+                .read_line(&mut sex)
+                .expect("Failed to read line");
+            let sex = sex.trim().parse::<u8>().unwrap();
+
+            if sex == 1 {
```

Comment:
let sex = if sex == 1 { Sex::Male } else {Sex::Female };
Initialise the student.